### PR TITLE
Card branch

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

File: components/utile.css [hunk 1]
```diff
@@ -1,3 +1,217 @@
+.head-1 {
+  font-size: 48px;
+  line-height: 64px;
+}
+
```
Comment (zoef-yt) [inline components/utile.css:5]: use rem for font size

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by dcs-soni — APPROVED** [review]
Looks good to me. 👍 Code structure is also good and class names are apt.

File: components/utile.css [hunk 2]
```diff
@@ -1,3 +1,217 @@
...
+.margin-xl {
+  margin: 2.5rem;
+}
+
+.margin-xxl {
+  margin: 3rem;
+}
+
+.padding-xxs {
+  padding: 0.25rem;
```
Comment (iRohitGaur) [inline components/utile.css:100]: for any value less than 10px, you should give in px

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: components/utile.css (hunk 2 above)
Comment (dev-prabhat) [reply components/utile.css:100]: okay

**Pull request merged**